pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -302,11 +302,6 @@ in
 
 assert lib.elem ffmpegVariant [ "headless" "small" "full" ];
 
-/*
- *  Licensing dependencies
- */
-assert withGPLv3 -> withGPL;
-assert withUnfree -> withGPL && withGPLv3;
```

Comment:
Oh, this was actually wrong. It should probably be re-instated as

```Nix
assert !(withUnfree && !withGPL);
```

as these are mutually exclusive.

Reply:
You can compile ffmpeg with both of them enabled, but the resulting binary becomes unredistributable, gone updated the license list to correctly reflect that